### PR TITLE
OR: new incidents

### LINKED ISSUE #847: Incident in Eugene, Oregon
--

## Location

Eugene, Oregon

## Date

may 30th

## Description
 
Police tear gas small group of peaceful protestors.

## Links

https://youtu.be/dPTr54DdTdY

https://www.reddit.com/r/Eugene/comments/gtbsvq/eugene_police_teargasses_downtown/?utm_medium=android_app&utm_source=share

### LINKED ISSUE #848: Incident in Eugene, Oregon
---

## Location

Eugene, Oregon

## Date

June 1st

## Description

Reporter shot with tear gas canister 

## Links
https://www.reddit.com/r/Eugene/comments/guxme1/footage_of_epd_shooting_eugene_weekly_reporter/?utm_medium=android_app&utm_source=share
https://youtu.be/R7fqWI41vQY
https://www.dailyemerald.com/news/police-fire-pellets-tear-gas-at-late-night-protesters-sunday-police-chief-responds/article_cd4a4450-a49a-11ea-a6fd-73b64efe150a.html

### LINKED ISSUE #854: Incident in Portland, Oregon
---

## Location

Portland, Oregon

## Date

July 8th

## Description

Police raid park vigil for slain protestor.

Protestors held a vigil in a park for a protestor killed by a driver several days prior. Police and federal agents arrived and deployed flashbangs. Officers then retreated to federal property, shooting riot control rounds at protestors as they did so.

## Links

https://twitter.com/hungrybowtie/status/1280758410602938370
https://twitter.com/hungrybowtie/status/1280759078826815489
https://twitter.com/hungrybowtie/status/1280775090582380545

### PR DESCRIPTION
**Pull request by emarcey**
Closes #847 
Closes #848
Closes #854

### CODE REVIEW

File: reports/Oregon.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
 ## Eugene
 
+### Peaceful protestor tear gassed | May 30th
+
+Footage shows a small cluster of protestors chanting "I can't breathe." At 1:15, without warning, police fire tear gas. Most protestors flee. Some canisters can be seen thrown back towards police.
```
Comment (alexkeating) [inline reports/Oregon.md:5]: Maybe make the second sentence  "Footage shows a small cluster of protestors chanting "I can't breathe." At 1:15, without warning, police fire tear gas causing most protestors to flee."

File: reports/Oregon.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+
+Police then proceed down the street. More tear gas is fired at 4:30.
```
Comment (alexkeating) [inline reports/Oregon.md:7]: Maybe add am/pm

File: reports/Oregon.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
...
+
+**Links**
+
+* https://www.youtube.com/watch?v=dPTr54DdTdY
+* https://www.reddit.com/r/Eugene/comments/gtbsvq/eugene_police_teargasses_downtown/
+
+
+### Reporter shot with tear gas canister | May 31st
+
+Footage shows police giving curfew dispersal order to protestors. At one point, the filmer and his fellow journalist identify themselves as a reporter. One officer says it doesn't matter, they need to leave. They continue to monitor the protest and are later joined by a reporter for the Eugene Weekly.
```
Comment (alexkeating) [inline reports/Oregon.md:21]: Is "it doesn't matter, they need to leave" a quote?

File: reports/Oregon.md (hunk 2 above)
Comment (emarcey) [reply reports/Oregon.md:7]: oh this is the timestamp but I'll make it clearer

File: reports/Oregon.md (hunk 3 above)
Comment (emarcey) [reply reports/Oregon.md:21]: It's a pseudo-quote I'll make it clearly not one

**Pull request merged**